api/api/utils/image_proxy/__init__.py

```diff
@@ -192,28 +190,10 @@ async def get(
         exception_name = f"{exc.__class__.__module__}.{exc.__class__.__name__}"
         key = f"thumbnail_error:{exception_name}:{domain}:{month}"
         try:
-            count = await tallies_incr(key)
+            await tallies_incr(key)
         except ConnectionError:
             logger.warning("Redis connect failed, thumbnail errors not tallied.")
-            # We will use a counter to space out Sentry logs.
-            count = next(exception_iterator)
```

Comment:
We can also delete the `exception_iterator` definition on L29 (→L27) and then also the import for `itertools` on L1.

Reply:
Done in [2554558](https://github.com/WordPress/openverse/pull/3709/commits/2554558637150365326022bf2c5dce4ebc058708)!